src/content/apis/art/rijksmuseum.md

```diff
@@ -0,0 +1,10 @@
+---
+name: RijksData
```

Comment:
```suggestion
name: Rijks museum
```

Reply:
RijksData is the name of the site, but I changed this to the museum name. (The correct spelling is Rijksmuseum - one word.) Thanks!